custom/enikshay/case_utils.py

```diff
@@ -150,15 +145,12 @@ def get_associated_episode_case_for_test(test_case, occurrence_case_id):
 
 
 def get_all_episode_cases_from_person(domain, person_case_id):
-    case_accessor = CaseAccessors(domain)
-    episode_cases = []
     occurrence_cases = get_all_occurrence_cases_from_person(domain, person_case_id)
-    for occurrence_case in occurrence_cases:
-        all_cases = case_accessor.get_reverse_indexed_cases([occurrence_case.get_id])
-        episode_cases += [case for case in all_cases
-                          if case.type == CASE_TYPE_EPISODE and
-                          case.dynamic_case_properties().get('episode_type') == "confirmed_tb"]
-    return episode_cases
```

Comment:
Note that this change also switches from a query per occurrence_case to one query across all occurrence_cases under a person.